src/Pages/ProductPage/Product.jsx

```diff
@@ -0,0 +1,97 @@
+import "./product.css";
+import "./product-Responsive.css";
+import { Nav } from "../../components/Nav/Nav";
+import { NavForMobile } from "../../components/NavForMobile/NavForMobile";
```

Comment:
good to see that you have created components for mobile view also.

Reply:
ThankYou